projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad.php

```diff
@@ -576,6 +588,30 @@ function wpcom_launchpad_is_keep_building_enabled() {
 	return false;
 }
 
+/**
+ * Checks if the Blog flow task list is enabled.
+ *
+ * @return bool True if the task list is enabled, false otherwise.
+ */
+function wpcom_launchpad_is_intent_write_enabled() {
+
+	if ( ! is_automattician() ) {
```

Comment:
I don't think we can use the `is_automattician()` check here; it may not be available on Atomic sites.

Reply:
Here is the related diff so we do the check on a filter: D115777-code